.github/workflows/parachain.yml

```diff
@@ -140,3 +145,43 @@ jobs:
           --manifest-path cumulus/Cargo.toml 
           --release --verbose 
           --package bridge-hub-rococo-runtime
+
+  beacon-fuzz:
+    needs: test
```

Comment:
Alternatively I can make this part of the tests job, so only one checkout and build is necessary.

Reply:
Would prefer a seperate job as it seems will require a long time.

Reply:
In addition to the time taken, fuzz tests are intended to find new failing cases so separate jobs & statuses for unit vs fuzz tests makes sense. We always want unit tests to pass, but a fuzz test failure might warrant a separate task.

We can always cache the results of any shared work between regular & fuzz tests to improve performance.